test/one/profiler/test/Runner.java

```diff
@@ -149,7 +150,11 @@ private static boolean run(Method m) {
             String testLogDir = logDir.isEmpty() ? null : logDir + '/' + testName;
             try (TestProcess p = new TestProcess(test, currentOs, testLogDir)) {
                 Object holder = (m.getModifiers() & Modifier.STATIC) == 0 ? m.getDeclaringClass().newInstance() : null;
-                m.invoke(holder, p);
+                if (p.inputs().length == 0) {
+                    m.invoke(holder, p);
+                } else {
+                    m.invoke(holder, p, p.inputs());
```

Comment:
I'm not sure we need to pass `inputs()` explicitly.
Test method already accepts `p` and can extract inputs whenever it wants.